test/integ/hostOnlyAPI/src/hostOnlyAPI.cpp

```diff
@@ -0,0 +1,185 @@
+/* Copyright 2022 Andrea Bocci
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// make sure the CPU_B_SEQ_T_SEQ backend is always available
```

Comment:
Is this for your CMS build system? CMake should enable the sequential back-end by default if the test cases are enabled.

Reply:
Mhm, no, we should build the tests with the native CMake.

I guess I misunderstood something I read, that `ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED` was not guaranteed to always be available. I'll remove the definition.

Reply:
By default it isn't, but we force-enable it if the examples and/or tests are switched on through CMake.

Reply:
OK.

Reply:
Mhm, it doesn't seem to always be the case, see https://github.com/alpaka-group/alpaka/runs/4853400473?check_suite_focus=true#step:4:2982 :
```
D:\a\alpaka\alpaka\test\integ\hostOnlyAPI\src\hostOnlyAPI.cpp(95,29): error C2039: 'AccCpuSerial': is not a member of 'alpaka' [D:\a\alpaka\alpaka\build\test\integ\hostOnlyAPI\hostOnlyAPITest.vcxproj]
```

Reply:
Is it because this is compiled as a standard (host-only) application, instead of an "alpaka" application ?